example/lib/main.dart

```diff
@@ -19,6 +19,7 @@ class ShimmerPage extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       body: Shimmer(
+//        duration: Duration(seconds: 2),
```

Comment:
`//        duration: Duration(seconds: 3),    //default value`

Since this is the default value that would be used.

Reply:
Let it be. Never mind